src/content/docs/usage/workspaces.mdx

```diff
@@ -125,6 +125,103 @@ You can set the Project Builder at Workspace creation time using command-line
 flags. Refer to **[Builders](/docs/usage/builders)** for more information.
 </Aside>
 
+## Create a Workspace with Multiple Projects
+
+Daytona allows you to create a Workspace with multiple projects, enabling you to work on multiple codebases simultaneously within a single environment.
```

Comment:
I would add an example of what this means.

e.g. When using the DO provider, multiple projects will be spun up inside the same droplet, AWS - EC2, etc.

Reply:
`The [Docker Provider](/docs/configuration/providers#docker) deploys projects within the same container.`

This is not correct. There is no workspace in the Docker Provider. Each project is its own container. The "workspace" is the host machine.